lib/storage/http.js

```diff
@@ -51,13 +42,12 @@ async function upload(rendition) {
     }
 
     try {
-        const contentType = mime.contentType(path.extname(file)) || DEFAULT_CONTENT_TYPE;
         if (typeof target === 'string') {
             console.log(`uploading rendition ${file} to ${AssetComputeLogUtils.redactUrl(target)}, size = ${rendition.size()}`);
             await http.uploadFile(file, target, {
                 retryEnabled: !process.env.ASSET_COMPUTE_DISABLE_RETRIES,
                 headers: {
-                    'content-type': contentType
+                    'content-type': rendition.mimeType()
```

Comment:
This should use `rendition.contentType()` instead to include `charset` in case of text files), which was done by `mime.contentType()` used before.